constantine/commitments_setups/README.md

```diff
@@ -1,6 +1,6 @@
-# Trusted Setups
+# Reference Strings & Trusted Setups
```

Comment:
With IPA, we now need to support transparent setups, CRS / URS for Common/Universal Reference Strings. Those are opposed to SRS, (hidden) structured reference strings, which have a toxic secret that no one should have access to, i.e. a trusted setup.

The folder likely should be named protocol_setups but since those setups are related to the polynomial commitments, it's nice to have both close to each other